modules/gapi/test/cpu/gapi_core_tests_fluid.cpp

```diff
@@ -142,6 +142,9 @@ INSTANTIATE_TEST_CASE_P(ConvertToFluid, ConvertToTest,
                                        cv::Size(1280, 720),
                                        cv::Size(640, 480),
                                        cv::Size(128, 128)),
+                                Values(3.0, 1.0, 0.5, 0.0, -1.0),
+                                Values(256.0, 32.0, 0.0, -128.0),
+                                Values(Tolerance_FloatRel_IntAbs(1e-5, 2).to_compare_f()),
```

Comment:
Fluid version is the only reason for tolerance. Seems like we have minor round issues with CV_32FC1 and alpha = 3, beta = {256, -128}